src/commands/flutter-symbols/README.md

```diff
@@ -0,0 +1,53 @@
+## Overview
+
+To deobfuscate and symbolicate errors and crashes, upload your Flutter Symbols, iOS dSYMs, and Android mapping Files to Datadog
+
+## Setup
+
+You need to have `DATADOG_API_KEY` in your environment.
+
+```bash
+# Environment setup
+export DATADOG_API_KEY="<API KEY>"
+```
+
+You can configure the tool to use Datadog EU by defining the `DATADOG_SITE` environment variable as `datadoghq.eu`. By default, the requests are sent to Datadog US.
```

Comment:
```suggestion
In order to configure the tool to use Datadog EU, define the `DATADOG_SITE` environment variable as `datadoghq.eu`. By default, requests are sent to Datadog US.
```

Reply:
Yeah, I was wondering about this. @louiszawadzki we may want to make this change in the React Native readme as well.